sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -347,13 +328,11 @@ def properties(self):
         return self._properties
 
     @properties.setter
-    def properties(self, value):
-        # type: (AmqpMessageProperties) -> None
+    def properties(self, value: "AmqpMessageProperties") -> None:
```

Comment:
For Python 3.7+, you can add `from __future__ import annotations` at the top of the file and remove the forward reference here:
```suggestion
    def properties(self, value: AmqpMessageProperties) -> None:
```